README.md

```diff
@@ -52,15 +52,39 @@ We provide several notebooks to show how recommendation algorithms can be design
 
 - The [Operationalizion Notebook](notebooks/05_operationalize) demonstrates how to deploy models in production systems.
 
+
+The Quick-Start and Modeling notebooks showcase how to utilize the following algorithms to build a recommender system:
+
+**Algorithms**
+
+| Algorithm | Environment | Description | 
```

Comment:
Ideally in the description we should put 1 sentence overviews of the strengths and weaknesses of the algorithm, and where the algorithm should be used. This description should NOT just describe the algorithm and dataset we're using it on, but also describe when and where a user should use this algorithm (where does it shine?),

Reply:
I think that makes sense